scour/scour.py

```diff
@@ -3433,6 +3433,42 @@ def choose_quote_character(value):
                          'flowDiv', 'flowPara', 'flowSpan', 'flowTref', 'flowLine']
 
 
+KNOWN_ATTRS = [
+        # TODO: Maybe update with full list from https://www.w3.org/TR/SVG/attindex.html
+        # (but should be kept intuitively ordered)
+        'id', 'xml:id', 'class',
+        'transform',
+        'x', 'y', 'z', 'width', 'height', 'x1', 'x2', 'y1', 'y2',
+        'dx', 'dy', 'rotate', 'startOffset', 'method', 'spacing',
+        'cx', 'cy', 'r', 'rx', 'ry', 'fx', 'fy',
+        'd', 'points',
+    ] + sorted(svgAttributes) + [
+        'style',
+    ]
+
+KNOWN_ATTRS_ORDER_BY_NAME = defaultdict(lambda: len(KNOWN_ATTRS),
+                                        {name: order for order, name in enumerate(KNOWN_ATTRS)})
+
+
+# use custom order for known attributes and alphabetical order for the rest
+def _attribute_sort_key_function(attribute):
+    name = attribute.name
+    order_value = KNOWN_ATTRS_ORDER_BY_NAME[attribute.name]
+    return order_value, name
+
+
+def attributes_ordered_for_output(elem):
+    if not elem.hasAttributes():
+        return []
+    attr = elem.attributes
```

Comment:
Personal preference (i.e. no need to fix if doing so would annoy you 😉): Use full names, e.g.
* `element` instead of `elem`
* `attributes = element.attributes` instead of `attr = elem.attributes`

Especially when refactoring code, it's usually a good opportunity to implement such changes also to existing code.

Reply:
Done